app/src/main/kotlin/com/bitwarden/authenticator/ui/authenticator/feature/itemlisting/ItemListingViewModel.kt

```diff
@@ -528,6 +534,17 @@ class ItemListingViewModel @Inject constructor(
         sendEvent(ItemListingEvent.NavigateToBitwardenListing)
     }
 
+    private suspend fun handleDropdownMenuClick(action: ItemListingAction.DropdownMenuClick) {
+        sendAction(
+            when (action.menuAction) {
+                VaultDropdownMenuAction.COPY -> ItemListingAction.ItemClick(action.id)
```

Comment:
We do not need to send actions like this, just call the functions (no need to suspend).

I am also wondering if wee need the actions at all?